Formula/mesa.rb

```diff
@@ -47,6 +47,7 @@ class Mesa < Formula
   uses_from_macos "zlib"
 
   on_linux do
+    depends_on "gcc"
```

Comment:
Since you already have an `llvm` dependency, wouldn't it be simpler to build with `clang` instead, so you don't need an extra dependency?

Reply:
See https://github.com/Homebrew/linuxbrew-core/pull/23230#issuecomment-840079673

Sorry I forgot to add this to the PR description, I was a little bit in a hurry.

The mesa build is a little bit messy regarding the llvm / gcc situation and this is the only way I found to let the build finish.

Reply:
Not sure you discuss building with LLVM `clang` in that PR?

Reply:
We tried llvm alone, see the log: https://github.com/Homebrew/linuxbrew-core/pull/23230#issuecomment-840075394

(using `fails_with :gcc`) and for llvm usage. But mesa was sort of unhappy about that. I force pushed the PR, so you don't see the initial attempt anymore to build it with llvm.

In general we also prefer to stick with gcc (5 or 10) instead of relying on llvm.